src/commands/import.ts

```diff
@@ -87,8 +107,24 @@ export default class Import extends Kommand {
         const total = await restoreProfiles(this, dump)
 
         this.logOk(`[profiles] Imported ${total} profiles`)
+      } catch (error) {
+        this.logKo(`Error during import of ${file}: ${error.message}. Skipped.`)
       }
-      catch (error) {
+    }
+    if (files.userMapping.length !== 0) {
```

Comment:
You should follow the same logic with the  `for` loop. Even if there is little chances that someone import 2 users mappings, it could still happen.

Reply:
Not sure of the comportement needed here.
If someone has two users mappings it means that he want both to be imported ? 
We might throw or skip the mapping import and display something like `There are multiple users mappings, you can import the one you want using following command kourou users:importMapping {filepath}`

Reply:
Multiple users mappings can be imported from different files as long as they don't define the same field twice but with a different type.

But seriously I doubt someone will do this actually :sweat_smile: This comment was mostly to have a uniform code